docs/game_sdk/Store.md

```diff
@@ -0,0 +1,449 @@
+
+## OnEntitlementDelete
+
+Fires when the connected user loses an entitlement, either by expiration, revocation, or consumption in the case of consumable entitlements.
+
+###### Parameters
+
+| name        | type                | description                       |
+| ----------- | ------------------- | --------------------------------- |
+| entitlement | Discord.Entitlement | the entitlement the user has lost |
+
+## HTTP APIs
+
+The following are HTTP requests, and should be handled by your game server, rather than a client. They require a Bearer token for an authorization header. This token should be a token of a developer who is authorized to use Dispatch for your application. To get this token, your backend service should go through the [Client Credentials OAuth2 Grant](#DOCS_TOPICS_OAUTH2/client-credentials-grant) and request the scope `applications.entitlements`.
+
+## Get Entitlements % GET /applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements
+
+Gets a entitlements for a given user. You can use this on your game backend to check entitlements of an arbitrary user, or perhaps in an administrative panel for your support team.
+
+###### Query Parameters
+
+| name    | type                        | description                                           |
+| ------- | --------------------------- | ----------------------------------------------------- |
+| user_id | int                         | the user id to look up entitlements for               |
+| sku_ids | comma-separated list of int | (optional) the list SKU ids to check entitlements for |
+
+###### Example
+
+```
+curl https://discordapp.com/api/v6/applications/461618159171141643/entitlements?user_id=53908232506183680 \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  [
+    {
+      "user_id": "53908232506183680",
+      "sku_id": "53908232599983680",
+      "application_id": "461618159171141643",
+      "id": "53908232506183999",
+      "type": 3
+    }
+  ]
+}
+```
+
+## Get Entitlement % GET /applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements/{entitlement.id#DOCS_GAME_SDK_STORE/data-models-entitlement-struct}
+
+Fetch an entitlement by its ID. This may be useful in confirming that a user has a given entitlement that another call or the SDK says they do.
+
+```
+curl https://discordapp.com/api/v6/applications/461618159171141643/entitlements/53908232506183999 \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  "user_id": "53908232506183680",
+  "sku_id": "53908232599983680",
+  "application_id": "461618159171141643",
+  "id": "53908232506183999",
+  "type": 3
+}
+```
+
+## Get SKUs % GET /applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/skus
+
+Get all SKUs for an application. You can filter the type of SKUs returned via the optional query parameter.
+
+###### Query Parameters
```

Comment:
this documentation is mixed up. The endpoint URL is correct. It does not take any filter parameters though (the public `/store` endpoint does).

this is my fault for mixing you up here, wasn't aware that `store` is internal only...